### ISSUE TITLE
Restore GPL licensing option

### ISSUE THREAD

**Issue by luke-jr** [pravins/lohit#88]
I don't consider OFL a free font license due to the no-sales clause. Can you dual license it under GPL (like before) and OFL?

**Comment by pravins** [comment]
GPL license never created specific for the fonts and we were using exception clause with GPL.. while OFL licence is specifically made for the fonts.

Can you copy paste exact objectionable clause of OFL here?

**Comment by davelab6** [comment]
Section 1 says you can't sell the work "by itself", but fsf, OSPO, Debian have accepted this as free. The OFL what requires redistribution only under itself, so dual licensing is bad because anyone downstream can't also dual license, and if there are multiple copyright holders, no single once can change the license.

**Comment by luke-jr** [comment]
"1) Neither the Font Software nor any of its individual components, in Original or Modified Versions, may be sold by itself."

The common argument is that you can just sell a dummy program with it. But I don't think that would hold up in court.

Therefore, it is not a free license.

**Comment by davelab6** [comment]
Are you a lawyer?

**Comment by luke-jr** [comment]
Nope, just a free software advocate who happens to maintain a list of free fonts and would prefer not to have to drop this one.
